docs/features/telemetry-enrichment.md

```diff
@@ -22,4 +22,25 @@ ILogger logger = new LoggerConfiguration()
     .CreateLogger();
 
 logger.Information("This event will be enriched with the runtime assembly product version");
+```
+
+## Kubernetes Enricher
+
+The `Arcus.Observability.Telemetry.Serilog` library provides a [Kubernetes](https://kubernetes.io/) enricher that adds several machine information from the environment (variables).
+
+**Example**
+| Environment Variable   | Log Property | Value                |
+| ---------------------- | ------------ | -------------------- |
+| `KUBERNETES_NODE_NAME` | NodeName     | `spec.nodeName`      |
```

Comment:
Can we get rid of the value column please, it depends on the configuration.
I'll send a PR with an example so once it's merged you can assign the issue to me

Reply:
Done, and I will.